src/lib/containers/EntityBadge.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react'
```

Comment:
Not a 100% drop-in replacement for the SWC `EntityBadge.java`, because this component does not display
* An 'unlink' button, when applicable
* MD5
* Content size
* Add to download list
* Perhaps other functionality

It does display the list of icons though. I think the other functionality should probably be handled by other components in a React paradigm